models/aether-4.x/openapi.yaml

```diff
@@ -292,7 +292,7 @@ components:
           type: string
         endpoint-id:
           description: Id of this endpoint
-          maxLength: 9223372036854775808
+          maxLength: 18446744073709551615
```

Comment:
MacOS on M1 gives a different result for this than linux.
It used to fail Jenkins for me - seems to be passing now for you